addons/im_livechat/controllers/chatbot.py

```diff
@@ -63,22 +62,18 @@ def chatbot_trigger_step(self, channel_id, chatbot_script_id=None):
             return None
 
         posted_message = next_step._process_step(discuss_channel)
-        return {
-            "data": Store(posted_message, for_current_user=True).get_result(),
-            'scriptStep': {
-                'id': next_step.id,
-                'answers': [{
-                    'id': answer.id,
-                    'label': answer.name,
-                    'redirectLink': answer.redirect_link,
-                } for answer in next_step.answer_ids],
-                'isLast': next_step._is_last_step(discuss_channel),
-                'message': plaintext2html(next_step.message) if not is_html_empty(next_step.message) else False,
-                'type': next_step.step_type,
+        store = Store()
+        store.add(
```

Comment:
single .add can be Store params directly and `get_result` and be chained (if it helps)